packages/dart/noports_core/lib/src/sshnpd/sshnpd_impl.dart

```diff
@@ -165,7 +179,7 @@ class SshnpdImpl with AtClientBindings implements Sshnpd {
         throw ('\n Unable to find .atKeys file : ${p.atKeysFilePath}');
       }
 
-      AtSignLogger.root_level = 'SHOUT';
+      AtSignLogger.root_level = 'SEVERE';
```

Comment:
For the daemon, SEVERE is more appropriate than SHOUT as the default, so people see SEVERE errors in their logs, which, ideally, they should